src/ResourceManager/Cdn/ChangeLog.md

```diff
@@ -17,7 +17,7 @@
     * Overview of change #1
         - Additional information about change #1
 -->
-## Current Release
+## Current Release (Add disable and enable CDN Custom domain Https)
 
```

Comment:
See the examples above.  This should include a bullet for each added cmdlet, i.e.
* Added cmdlets to manage custom domains
  * Disable-AzureRmCdnCustomDomain
  * Remove-AzureRmCdnCustomDomain